app/modals/stacking/stacking-modal.tsx

```diff
@@ -267,7 +267,7 @@ export const StackingModal: FC<StackingModalProps> = ({ onClose, numCycles, poxA
             isDisabled={isDecrypting}
             onClick={() => broadcastTx()}
           >
-            Send transaction
+            Lock STX
```

Comment:
Should we perhaps say "Initiate Stacking" since we'll say "Initiating Stacking..." soon after in the activity?